.gitignore

```diff
@@ -106,10 +106,10 @@ ENV/
 # Mac OS temporary file
 .DS_Store
 
-# Drucker
-drucker_pb2.py
-drucker_pb2_grpc.py
+# Rekcurd
+rekcurd_pb2.py
+rekcurd_pb2_grpc.py
 db.test.sqlite3
 test-kube-config/
-tests/drucker_dashboard/models/
+tests/rekcurd_dashboard/models/
```

Comment:
not sure if we still need them because there is no submodules any more

Reply:
Thank you @yuki-mt 
I have removed unnecessary lines. -> https://github.com/rekcurd/community/pull/19/commits/f2472658e165e40a050020ef53e9984cc46303a3